dtls.c

```diff
@@ -1,6 +1,6 @@
 /*******************************************************************************
  *
- * Copyright (c) 2011-2021 Olaf Bergmann (TZI) and others.
+ * Copyright (c) 2011-2022 Olaf Bergmann (TZI) and others.
```

Comment:
Not really wrong, but I would prefer to have that listed for the code-cleanup, issue #143

Reply:
Fixed I wrongly rebased to main which had pulled in unrelated changes.

Reply:
Since this PR had been opened against develop I kept it that way, or should I re-open against main?

Reply:
FMPOV, that's a detail of issue #140. 
For now it's a "raw idea", to have a "main" and a "develop", mainly to relax the time constraints for team members. Though this project works different with git as I'm used to it, I'm not sure, if my way will the right one. (My way would be, PR against "main", cherrypick in advance to "develop", if indicated by time constraints.)
Anyway, though this PR is dated before #140, I don't think, it's required to recreate a PR against main.

Reply:
Makes sense, so we can move forward with this one as is.